discord/permissions.py

```diff
@@ -544,7 +544,7 @@ def use_external_stickers(self) -> int:
 
 PO = TypeVar('PO', bound='PermissionOverwrite')
 
-def _augment_from_permissions(cls):
+def _augment_from_permissions(cls: Type[PO]) -> Type[PO]:
```

Comment:
```suggestion
def _augment_from_permissions(cls):
```

Please don't. This is annoying to type properly in pyright in my experience and the inference works fine.